docs/model_developers/handlers.rst

```diff
@@ -233,28 +233,28 @@ method annotated with this annotation will be cached, similar to the way
 `lru_cache <https://docs.python.org/3/library/functools.html#functools.lru_cache>`_ works. The arguments to
 the method will form the cache key, the return value will be cached. When the method is called a
 second time with the same arguments, it will not be executed again, but the cached result is
-returned instead. To exclude specific arguments from the cache key, use the `ignore` parameter.
+returned instead. To exclude specific arguments from the cache key, use the ``ignore`` parameter.
 
-Cache entries will be dropped from the cache when they become stale.
-The for_version and timeout parameters control this behaviour:
+Cache entries will be dropped from the cache when they become stale. Use the following parameters to control when an entry is considered stale:
+  * Setting ``evict_after_creation=True`` will mark entries as stale after a timeout controlled by the ``timeout`` parameter (5000s by default).
+  * Setting ``refresh_after_access=True`` will reset the expiry time of entries to 60s anytime they're used.
```

Comment:
Perhaps it was already discussed, and I'm not aware of the conclusion of the discussion, but don't we want to be a bit more flexible than the 60s: be able to set any timeout for this kind of cache as well?